net/src/memory_server.rs

```diff
@@ -1,20 +1,31 @@
 //! provides fake in-memory p2p worker for use in scenario testing
 
-use holochain_core_types::cas::content::Address;
+#![allow(non_snake_case)]
```

Comment:
ditto snake case question

Reply:
ah, in this file i do see non-snake-case symbols though : )

Reply:
need it for `priv_serve_HandleGetHoldingMetaListResult()`